astronomer/providers/apache/livy/example_dags/example_livy.py

```diff
@@ -160,6 +154,11 @@ def add_inbound_rule_for_security_group(task_instance: Any) -> None:
                 }
             ],
         )
+    except ClientError as error:
+        if error.response.get("Error", {}).get("Code", "") == BOTO_DUPLICATE_PERMISSION_ERROR:
+            logging.info("Ingress for port 22 already authorized")
```

Comment:
It has to be exception instead of info ?

Reply:
Since, we're catching the exception and is known I put it to info. We can discuss this with the team on what the log level could be here.

Reply:
I guess I will put it log level ERROR instead of EXCEPTION since we don't need the stacktrace here